commands/RunFor.java

```diff
@@ -21,38 +22,35 @@
 	 *        Whether this command should be interruptible
 	 */
 	public RunFor(Command command, double duration) {
-		super("RunFor[" + command.getName() + "]");
 		this.duration = duration;
 		this.command = command;
 		firstTick = true;
-		setTimeout(duration);
```

Comment:
add in `addRequirements(this.command.getRequirements)`

Reply:
`this.command.getRequirements()` returns an `Set<Subsystem>`. How should I pass that to an arglist?

Reply:
fixing this issue now by using `addRequirements((Subsystem[]) this.command.getRequirements().toArray())`.